lib/elixir/lib/io/ansi.ex

```diff
@@ -29,6 +29,12 @@ defmodule IO.ANSI do
   @typep ansilist :: maybe_improper_list(char() | ansicode() | binary() | ansilist(), binary() | ansicode() | [])
   @type  ansidata :: ansilist() | ansicode() | binary()
 
+  defmacro valid_rgb216(r, g, b) do
```

Comment:
That's a good point.  I originally wrote the validation as a macro to use it as a guard clause, but looking at it again, the logic is simple enough that a macro is not really justified.  I can fix that up and submit another PR.

As a side question: I see that the defsequence macro is defined in a separate module and required in IO.ANSI-- is this what keeps users from having to `require IO.ANSI`?